docker/Dockerfile

```diff
@@ -85,7 +85,7 @@ RUN mkdir /etc/service/apache2 && \
 	chmod +x /etc/service/apache2/run
 
 # Add the harvesting cronjob (once per day)
-RUN (crontab -l ; echo "0 1 * * * source /var/www/stationlite/venv3/bin/activate && \
+RUN (crontab -l ; echo "0 1 * * * . /var/www/stationlite/venv3/bin/activate && \
```

Comment:
@Jollyfant,

this fix IMO won't work. According to the bash (1) man pages the `.` and `source` builtins are the same:

```
$ man 1 bash
[...]
        .  filename [arguments]
       source filename [arguments]
              Read  and  execute commands from filename in the current shell environment and return the exit status of the last command executed from filename.  If filename does not contain a slash, filenames in PATH are used to find
              the directory containing filename.  The file searched for in PATH need not be executable.  When bash is not in posix mode, the current directory is searched if no file is found in PATH.  If the sourcepath option to  the
              shopt builtin command is turned off, the PATH is not searched.  If any arguments are supplied, they become the positional parameters when filename is executed.  Otherwise the positional parameters are unchanged.  If the
              -T option is enabled, source inherits any trap on DEBUG; if it is not, any DEBUG trap string is saved and restored around the call to source, and source unsets the DEBUG trap while it executes.  If -T is  not  set,  and
              the sourced file changes the DEBUG trap, the new value is retained when source completes.  The return status is the status of the last command exited within the script (0 if no commands are executed), and false if file‐
              name is not found or cannot be read.
[...]
```

Instead I'd recommend using something like
```
RUN (crontab -l ; echo '0 1 * * * bin-bash -c "source /var/www/stationlite/venv3/bin/activate && \
     python /var/www/mediatorws/eidangservices/stationlite/harvest/harvest.py \
     sqlite:////var/www/mediatorws/db/stationlite.db"') | crontab
```
or even better
```
RUN (crontab -l ; echo "0 1 * * * \
     /var/www/stationlite/venv3/bin/eida-stationlite-harvest \
     sqlite:////var/www/mediatorws/db/stationlite.db") | crontab
```

i.e. let setuptools with its `entry_points` do the job for you.

Reply:
Ah thanks, @Jollyfant. Yes, use:

```
RUN (crontab -l ; echo "0 1 * * * \
     /var/www/stationlite/venv3/bin/eida-stationlite-harvest \
     sqlite:////var/www/mediatorws/db/stationlite.db") | crontab
```

and we should be fine. No `.` vs `source` issues anymore.